spec/system/task_spec.rb

```diff
@@ -1,24 +1,25 @@
 require 'rails_helper'
+require 'byebug'
 
 RSpec.describe 'Task', type: :system do
+  let!(:project) { create(:project) }
+  let(:task) { create(:task, project_id: project.id) }
+  let(:done_task) { create(:task, :done_task, project_id: project.id) }
+
   describe 'Task一覧' do
     context '正常系' do
       it '一覧ページにアクセスした場合、Taskが表示されること' do
-        # TODO: ローカル変数ではなく let を使用してください
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id)
         visit project_tasks_path(project)
         expect(page).to have_content task.title
         expect(Task.count).to eq 1
         expect(current_path).to eq project_tasks_path(project)
       end
 
-      xit 'Project詳細からTask一覧ページにアクセスした場合、Taskが表示されること' do
-        # FIXME: テストが失敗するので修正してください
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id)
+      it 'Project詳細からTask一覧ページにアクセスした場合、Taskが表示されること' do
         visit project_path(project)
         click_link 'View Todos'
+        handle = page.driver.browser.window_handles.last
+        page.driver.browser.switch_to.window(handle)
```

Comment:
1行で書けるのでこっちの方が良いかもですね。
```suggestion
        switch_to_window(windows.last)
```

Reply:
0fa8abb で修正しました。